code/notificationbuilder/src/main/java/com/adobe/marketing/mobile/notificationbuilder/internal/builders/AutoCarouselNotificationBuilder.kt

```diff
@@ -137,6 +137,7 @@ internal object AutoCarouselNotificationBuilder {
                     trackerActivityClass,
                     R.id.carousel_item_image_view,
                     interactionUri,
+                    null,
```

Comment:
i noticed you had asked about providing the action id here in my pr. I have added an action id even if it is currently unused as it allows for a little more detailed tracking to be done without future changes to the push template code. i'm fine leaving it null for now though.

Reply:
Sounds good. I will add the rating selected here for this template. It could be passed in tracking data